walletnode/api/services/artwork.go

```diff
@@ -149,11 +151,61 @@ func (service *Artwork) Mount(ctx context.Context, mux goahttp.Muxer) goahttp.Se
 	return srv
 }
 
+// ArtSearch searches for artwork & streams the result based on filters
+func (service *Artwork) ArtSearch(ctx context.Context, req *artworks.ArtSearchPayload, stream artworks.ArtSearchServerStream) error {
+	defer stream.Close()
+	searchReq := fromArtSearchRequest(req)
+	taskID := service.search.AddTask(searchReq)
+	task := service.search.Task(taskID)
+
+	resultChan := task.SubscribeSearchResult()
+	for {
+		select {
+		case <-ctx.Done():
+			return nil
+		case srch, ok := <-resultChan:
```

Comment:
srch is search? if it's ok to use full name, please don't use the abbreviation, thank u